thoth/slo_reporter/sli_workflow_quality.py

```diff
@@ -101,37 +101,36 @@ def _report_sli(self, sli: Dict[str, Any]) -> str:
 
         for service in REGISTERED_SERVICES:
             service_metrics = {}
+
             for metric, value in sli.items():
+
                 if service in metric:
                     service_metrics[metric] = value
 
-            number_reports = sli[f"{service}_reports"]
-            number_workflow_failed = sli[f"{service}_workflows_failed"]
-            number_workflow_error = sli[f"{service}_workflows_error"]
+            number_workflows_succeeded = sli[f"{service}_workflows_succeeded"]
+            number_workflows_failed = sli[f"{service}_workflows_failed"]
+            number_workflows_error = sli[f"{service}_workflows_error"]
 
             if "ErrorMetricRetrieval" not in [v for v in service_metrics.values()]:
 
                 total_workflows = (
-                    int(number_reports)
-                    + int(number_workflow_failed)
-                    + int(number_workflow_error)
+                    int(number_workflows_succeeded) + int(number_workflows_failed) + int(number_workflows_error)
                 )
-                if int(number_reports) > 0:
+                if int(number_workflows_succeeded) > 0:
+
                     successfull_percentage = (
-                        (
-                            int(number_reports)
-                            - int(number_workflow_failed)
-                            - int(number_workflow_error)
-                        )
+                        (int(number_workflows_succeeded) - int(number_workflows_failed) - int(number_workflows_error))
```

Comment:
@pacospace  Should successful percentage be just `number_workflows_succeeded/total_workflows` * 100 ??
As this could lead to negative percentages in `successfull_percentage`.

Reply:
we can actually evaluate the total directly and have the percentage for succeeded, failed and error? WDYT?

Reply:
> we can actually evaluate the total directly and have the percentage for succeeded, failed and error? WDYT?

Yes that's what I was thinking, we are calculating the total count here - `total_workflows` we could just have the percentage of each, i.e. successful, error, failed, that could be useful.

Reply:
very nice catch!